_posts/2020-08-12-DDoS-criminal-hackers.md

```diff
@@ -0,0 +1,68 @@
+in the middle of the night presumably on behalf of the FBI. Homeland Security
+Investigations (HSI), one of the largest agencies within DHS, visited
+unaffiliated individuals hosting an archives of the Blue Leaks and even [offered them money](https://www.vice.com/en_us/article/xg84ww/ice-questions-admin-of-the-eye-archive-site-hosted-blue-leaks) to become informants. Just this week, documents obtained by Lucy Parsons Labs [confirm](https://cdn.muckrock.com/foia_files/2020/08/10/CCSO_00000003.pdf) DHS considers DDoS "criminal hackers" and not journalists.
+
+
+The government's aggressive push to name DDoS as "criminal hackers" is a
+dangerously unprecedented territory that criminalized everyone's right to
+publish online under the First Amendment. It is not illegal to possess and
+publish information obtained by a third party, even if it was obtained
+illegally, under the First Amendment as long as an individual does not
+participate in the illegal activity. This extends even to the publishing of
+classified information, as the Supreme Court
+[upheld](https://en.wikipedia.org/wiki/New_York_Times_Co._v._United_States) in
+the infamous case of the Pentagon Papers. (Almost all of the data published in
+the Blue Leaks was
+[unclassified](https://www.dla.mil/Portals/104/Documents/InformationOperations/LogisticsInformationServices/Customer%20Outreach/Subscriptions/J6_ForOfficialUseOnly_160113.pdf).)
+
+The confirmation of an investigation by DHS's Office of Intelligence and
+Analysis (I&A) reflect a growing patter of DHS investigating journalists as
+threats to national security. Just this month in Portland, I&A created "baseball cards" of
+[journalists](https://www.politico.com/news/2020/08/01/dhs-removes-official-who-oversaw-intelligence-reports-on-journalists-390042),
+a term most commonly used by the military in hostile conflict situations. Other DHS
+agencies have recently tracked
+[media](https://www.nbcsandiego.com/news/local/source-leaked-documents-show-the-us-government-tracking-journalists-and-advocates-through-a-secret-database/3438/)
+at the US-Mexico border in secret databases. While HSI is thought to be the
+government agency investigating DDoS, it is not uncommon for them to open
+investigations on behalf of other agencies such as the FBI in order for hide the
+existence of an official investigation. In the last few years, we have seen the
+FBI using [secret rules](https://theintercept.com/2016/06/30/secret-rules-make-it-pretty-easy-for-the-fbi-to-spy-on-journalists/)
+to open investigations on journalist through national security letters which do
+not require judicial review. As recently as 2018, the Freedom of the Press
+Foundation revealed a _seperate_ secret channel for opening an
+investigation on media organizations through [secret FISA court orders.](https://freedom.press/news/revealed-justice-depts-secret-rules-targeting-journalists-fisa-court-orders/)
```

Comment:
You forgot a period at the end of the sentence

Reply:
It's here `orders.]` I can move the hyperlink